products/backdrop.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Backdrop
+category: server-app
+tags: php-runtime
+permalink: /backdrop
+releasePolicyLink: https://backdropcms.org/releases
+releaseImage: https://backdropcms.org/files/images/release-cycles.png
+releases:
+-   releaseCycle: "1.29"
+    releaseDate: 2024-09-15
+    eol: false
+    latest: "1.29.1"
+    latestReleaseDate: 2024-10-06
+    link: https://github.com/backdrop/backdrop/releases/tag/1.29.1
```

Comment:
```suggestion
```
A `changelogTemplate` should be used instead of manually set the links. See next comment.